app/assets/javascripts/questions.coffee

```diff
@@ -1,3 +1,11 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+
+$(document).ready ->
+  $('input:radio').click ->
```

Comment:
good that you're using separate file
but this code will affect radio inputs for all application